src/modules/10-common/components/MultiTypeList/MultiTypeList.tsx

```diff
@@ -39,7 +39,7 @@ export interface MultiTypeListProps {
   multiTextInputProps?: Omit<MultiTextInputProps, 'name'>
   enableConfigureOptions?: boolean
   configureOptionsProps?: MultiTypeListConfigureOptionsProps
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
  formik?: FormikContextType<unknown>
```